langs/gleam/gleam.toml

```diff
@@ -1,5 +1,5 @@
 name = "code"
 
 [dependencies]
-gleam_stdlib = ">= 0.34.0 and < 2.0.0"
+gleam_stdlib = ">= 0.34.0 and < 0.44.0"
```

Comment:
Starting from version 0.44.0 of the standard library, Gleam returns a ton of deprecation warnings.

![image](https://github.com/user-attachments/assets/e49c0f11-0f1b-47fa-b063-42d20962263e)

Setting the upper limit to one earlier version in `gleam.toml` yields the normal behavior.

![image](https://github.com/user-attachments/assets/42898071-65f5-49d2-ac67-9c0b96513436)

Reply:
I did try several things with the package suggestions, unfortunately to no avail.